lib/ramble/ramble/modifier.py

```diff
@@ -255,10 +255,13 @@ def modded_variables(self, app):
                 # var_str = app.expander.expansion_str(var)
                 # prev_val = app.expander.expand_var(var_str)
                 prev_val = app.variables[var]
+                if var in extra_vars:
```

Comment:
nit: maybe using the single-assignment-form is clearer?

```
if var in extra_vars:
    prev_val = extra_vars[var]
else:
    prev_val = app.variables[var]
```

Reply:
Yeah, good point. Fixed.